config/dictionary_strings.rb

```diff
@@ -2031,6 +2031,12 @@
 _("Middleware Deployments")
 # TRANSLATORS: en.yml key: dictionary.table.middleware_deployments
 _("Middleware Deployments")
+# TRANSLATORS: en.yml key: dictionary.table.middleware_datasource
+_("Middleware Datasource")
+# TRANSLATORS: en.yml key: dictionary.table.middleware_datasource (plural form)
+_("Middleware Datasources")
+# TRANSLATORS: en.yml key: dictionary.table.middleware_datasources
+_("Middleware Datasources")
```

Comment:
I doubt that we have to do anything in that file. My understanding is, that there is some tooling that goes over all the `_()`  and similar in the source which then creates this file and probably `*.po(t)` ones. @Fryguy ?